js/client-mainmenu.js

```diff
@@ -8,8 +8,11 @@
 			'keydown textarea': 'keyDown',
 			'keyup textarea': 'keyUp',
 			'click .username': 'clickUsername',
+			'click .header-username': 'clickUsername',
```

Comment:
This part can be reverted, then?

Reply:
Yeah it can be taken out.
I forgot to remove it.